storage/SystemInfo/CmdBtrfs.cc

```diff
@@ -92,9 +100,15 @@ namespace storage
 		while( it!=lines.end() && boost::contains( *it, "devid " ) )
 		{
 		    y2mil( "devs line:" << *it );
-		    string device = extractNthWord( 7, *it );
-		    if ( !boost::contains( device, "/dev" ) )  // Allow /sys/dev or /proc/devices
-			ST_THROW( ParseException( "Not a valid device name", device, "/dev/..." ) );
+
+		    Device device;
+
+		    extractNthWord(1, *it) >> device.id;
+
+		    device.name = extractNthWord(7, *it);
```

Comment:
when you touch it, can you please add example output as commend to the begging of this parse? It can really helps with all those word extraction to see what is each of that trying to parse.

Reply:
We have the testsuite with several examples.